include/ocpp/common/ocpp_logging.hpp

```diff
@@ -60,6 +60,7 @@ class MessageLogging {
     std::filesystem::path security_log_file;
     std::ofstream security_log_os;
     std::mutex output_file_mutex;
+    std::function<std::string(const std::string& message)> sanitize_message_callback;
```

Comment:
As discussed in the last WG meeting, lets drop this callback from this PR and just merge the write only variable logging changes. I'll follow up with a generic solution